Update personas document for issue #3100 (#3214)
raft-tech/TANF-app: docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md

### OFA analysts

> OFA analysts review, analyze, and report on STT’s TANF data that’s submitted through TDP. They also help STTs through the transmission process by offering support and guidance around data quality (including fatal/warning edits). Analysts in the Tribal TANF Division also play a direct role communicating submission status, resubmission requests, and various reports to Tribes. 

Reviewer: Please delete "(including fatal/warning edits)".

**Supporting Documentation**

- [Description of the OFA support mailbox](<https://hhsgov.sharepoint.com/:w:/r/sites/TANFDataPortalOFA-UserResearch/Shared%20Documents/User%20Research/OFA%20Resource%20Mailbox%20notes.docx?d=w2542a110040b44c1b82706a54c6ec9f2&csf=1&web=1&e=O5meXQ>) :lock:

### OFA system admins

OFA admins are members of the TANF Data Division. They manage access requests for the TANF data portal, help troubleshoot issues with submission, and broadly ensure that OFA analysts have what they need to do their work.

### Regional program specialists

> 

Reviewer: Please change grantee to STT under regional program specialist

## Personas

> Research up to this point has suggested a need for three working personas; an OFA Data Analyst (Grace), an OFA System Admin (Elliott), and an STT data analyst. While these don't reflect the full diversity of  roles, responsibilities, and team makeups on both the grantee and federal levels, the "jobs to be done" for each in the context of TDP have a lot of overlap. A single persona could also have many different combinations of permissions available to them in the data portal. 

Reviewer: ```suggestion
Research up to this point has suggested a need for three working personas: an OFA Data Analyst (Grace), an OFA System Admin (Elliott), and an STT data analyst. While these don't reflect the full diversity of  roles, responsibilities, and team makeups on both the grantee and federal levels, the "jobs to be done" for each in the context of TDP have a lot of overlap. A single persona could also have many different combinations of permissions available to them in the data portal. 
```

| Role                                | OFA Data Portal User                                         |
| Age                                 | 35                                                           |
| Location                            | Washington DC                                                |

> | Work Environment                    | Home and Office, Office                                      |

Reviewer: ```suggestion
| Work Environment                    | Home and Office|
```

| Delights                            | 1. "With TDP, I have a lot [of data] once the State submits the data the data gets processed already so that's really good. Now  I don't need to spend too much time with state if the  data wasn't processed." <br /> 2. "Having direct and up to date access to the data in the database" |
| Tools                               | Django CSV exports, Excel, SAS, email, reporting/analytics software |

### Elliott — OFA System Admin

| Grace                               |                                                              |
| :---------------------------------- | :----------------------------------------------------------- |
| Role                                | OFA Data Portal User, Viewing and Editing Permissions        |

> | Age                                 | 37                                                           |

Reviewer: Can Elliott be 46 or something?

Author: updated

> | Location                            | USA, Occassionally In Office DC                              |

Reviewer: ```suggestion
| Location                            | USA, Occasionally In DC Office                              |
```

| Work Environment                    | Home and Office, Desktop, and Government Furnished Equipment (GFE) |

> | Goals<br /><br /><br />             | 1. Ensure availability of features and components of the system <br />2. Work with regional OFA staff to confirm who should and shouldn't be granted access to the system<br />3. Access system logs to troubleshoot problems and remediate scans as needed <br />4. Provide access to OCIO staff for security auditing purposes <br />5. Monitoring emails, notifications and answering STT Questions |

Reviewer 2: ```suggestion
| Goals<br /><br /><br />             | 1. Ensure availability of features and components of the system <br />2. Work with regional OFA staff to confirm who should and shouldn't be granted access to the system<br />3. Access system logs to troubleshoot problems and remediate scans as needed <br />4. Provide access to OCIO (ACF-Tech) staff for security auditing purposes <br />5. Monitoring emails, notifications and answering STT Questions |
```

> | Pain Points<br /><br /><br /><br /> | 1. "Django doesn't have all the filters I need to efficiently use it for certain admin tasks" <br /> 2. "The system can sometimes log me out with no warning" <br/> 3. Accessing data in relational data bases is only possible by disconnecting the HHS network on site. <br/> 4. The general toolstack as an administrator can be difficult to use on a GFE <br/> 5. Requesting access to developer tools can take months at a time to get approved                           |

Reviewer 2: ```suggestion
| Pain Points<br /><br /><br /><br /> | 1. "Django doesn't have all the filters I need to efficiently use it for certain admin tasks" <br /> 2. "The system can sometimes log me out with no warning" <br/> 3. Accessing data from cloud.gov-brokered databases is only possible by disconnecting from the HHS network on site and VPN when remote. <br/> 4. The toolstack needed as an administrator can be difficult to use on a Windows GFE <br/> 5. Requesting access to developer tools can take months at a time to get approved                           |
```

| Delights                            | 1. "Django has become a tool that provides me a lot of insights about data in the system rather than solely being a user administration tool" <br/> 2. "Automated notifications have been convenient [so that I am aware of what is happening]" <br/> 3. "Being able to use an interface and see what [actions] users did" <br/> 4. "A point and click experience to see how the system is performing...I use the front end the most"|
| Tools                               | TDP Portal, Django CSV exports, Excel, SAS, email, reporting/analytics software, Microsoft Suite, Python, Axe Dev Tools |

> | Expectations | 1. TDP should help sys admin do things from an interface as much as possible <br /> 2. We have to have another way to troubleshoot and remediate issues |

Reviewer 2: ```suggestion
| Expectations | 1. TDP should help sys admin do things from an interface as much as possible <br /> 2. We need to have multiple ways to troubleshoot and remediate issues |
```

> To view a further in depth system of the interactions, primary tasks and touchpoints taken by OFA System Admins, please refer to the ecosystem map in the supporting documents below.

Reviewer: ```suggestion
To view a further in-depth system of the interactions, primary tasks and touchpoints taken by OFA System Admins, please refer to the ecosystem map in the supporting documents below.
```

- [Grantee Attributes](<https://hhsgov.sharepoint.com/:x:/r/sites/TANFDataPortalOFA-UserResearch/_layouts/15/WopiFrame2.aspx?sourcedoc=%7Be83f84f1-71a0-459f-80a4-0d39bc250be9%7D&action=view&cid=f2e656cb-9245-4f02-af6d-2d7e1db6573d>) :lock:
- [Working Journey Maps & Personas Mural](https://app.mural.co/t/officeoffamilyassistance2744/m/officeoffamilyassistance2744/1608238114372/191c53b8ef538838bc8c179daa238dd5c5dcc9e8) :lock:

> - [Ecosystem Map OFA System Admin](https://app.mural.co/t/raft2792/m/raft2792/1728311825176/74942929a28857c3ef1c401f0d07ccfe1073e882?sender=uf443e00cb4626b8bec157528)

Reviewer: Can you add the :lock: icon to the last link? 
```suggestion
- [Ecosystem Map OFA System Admin](https://app.mural.co/t/raft2792/m/raft2792/1728311825176/74942929a28857c3ef1c401f0d07ccfe1073e882?sender=uf443e00cb4626b8bec157528) :lock:
```
